diff --git a/node_modules/cozy-pouch-link/dist/files.js b/node_modules/cozy-pouch-link/dist/files.js
index aa7514a..25cb65f 100644
--- a/node_modules/cozy-pouch-link/dist/files.js
+++ b/node_modules/cozy-pouch-link/dist/files.js
@@ -33,7 +33,10 @@ var queryFileById = /*#__PURE__*/function () {
               fetchPolicy: defaultFetchPolicy,
               singleDocData: true
             };
-            return _context.abrupt("return", client.query((0, _cozyClient.Q)('io.cozy.files').getById(id), queryOpts));
+            return _context.abrupt("return", client.fetchQueryAndGetFromState({
+              definition: (0, _cozyClient.Q)('io.cozy.files').getById(id),
+              options: queryOpts
+            }));
 
           case 2:
           case "end":
diff --git a/node_modules/cozy-pouch-link/dist/jsonapi.js b/node_modules/cozy-pouch-link/dist/jsonapi.js
index fe63937..b26519b 100644
--- a/node_modules/cozy-pouch-link/dist/jsonapi.js
+++ b/node_modules/cozy-pouch-link/dist/jsonapi.js
@@ -247,7 +247,7 @@ var computeFileFullpath = /*#__PURE__*/function () {
 
           case 27:
             _yield$queryFileById = _context.sent;
-            parentDir = _yield$queryFileById.data;
+            parentDir = _yield$queryFileById && _yield$queryFileById.data;
 
             if (parentDir !== null && parentDir !== void 0 && parentDir.path) {
               _path = buildPathWithName(parentDir.path, file.name);
